spec/datadog/tracing/contrib/rails/support/base.rb

```diff
@@ -1,4 +1,6 @@
 require 'rails/all'
+# Loaded by the `bin/rails` script in a real Rails application
+require 'rails/command' if Rails::VERSION::MAJOR > 4
```

Comment:
Can we remove the conditional and move the line to

- `spec/datadog/tracing/contrib/rails/support/rails5.rb`
- `spec/datadog/tracing/contrib/rails/support/rails6.rb`

But not

- `spec/datadog/tracing/contrib/rails/support/rails4.rb`

Reply:
Great suggestion! Done.